packages/x-components/src/components/modals/base-events-modal-close.vue

```diff
@@ -18,16 +18,17 @@
   import BaseEventButton from '../base-event-button.vue';
 
   /**
-   * Component containing an event button that emits {@link XEventsTypes.UserClickedCloseX} when
-   * clicked. It has a default slot to customize its contents.
+   * Component containing an event button that emits
```

Comment:
just `contains`?

Reply:
I like `Component containing...`

Reply:
With the david's suggestion we can change the text and format it in just one line